src/sx-design/src/Button/Button.js

```diff
@@ -43,13 +49,21 @@ export default function Button(props: Props): React.Element<'button'> {
     });
   }
 
+  const handleOnClick = useCallback(
+    (event) => {
+      if (props.isDisabled !== true) {
+        props.onClick(event);
+      }
+    },
+    [props],
```

Comment:
If we do need it, the dependency array should probably only include the two attributes being used in the function.